README.md

```diff
@@ -62,7 +62,7 @@ The [community repository](https://github.com/pingcap/community) hosts all infor
 [<img src="docs/contribution-map.png" alt="contribution-map" width="180">](https://github.com/pingcap/tidb-map/blob/master/maps/contribution-map.md#tidb-is-an-open-source-distributed-htap-database-compatible-with-the-mysql-protocol)
 
 Contributions are welcomed and greatly appreciated. See
-[CONTRIBUTING.md](https://github.com/pingcap/community/blob/master/CONTRIBUTING.md)
+[CONTRIBUTING.md](https://github.com/pingcap/community/blob/master/contributors/README.md)
```

Comment:
I remember that the old link is mentioned by many repos (e.g. https://github.com/pingcap/tidb#to-start-developing-tidb). 

IMHO, a better way to change it is to recover the file `https://github.com/pingcap/community/CONTRIBUTING.md` and add a link to [CONTRIBUTING.md](https://github.com/pingcap/community/blob/master/contributors/README.md) inside it.